en/Extending Obsidian/Obsidian URI.md

```diff
@@ -21,140 +30,239 @@ The `action` parameter is the action that you would like to perform. Available a
 - `daily` to create or open your daily note.
 - `search` to open a search.
 
-
 > [!warning] Encoding
 > Ensure that your values are properly URI encoded. For example, forward slash characters `/` must be encoded as `%2F` and space characters must be encoded as `%20`.
-> 
- This is especially important because an improperly encoded "reserved" character may break the interpretation of the URI. [See here for details](https://en.wikipedia.org/wiki/Percent-encoding).
+>
+> This is especially important because an improperly encoded "reserved" character may break the interpretation of the URI. [See here for details](https://en.wikipedia.org/wiki/Percent-encoding).
+
+## Get an Obsidian URL
+
+To get an Obsidian URL for any note, use the **Copy Obsidian URL** command. This command is available in:
+
+- The [[Command palette]]
+- The three-dot menu in the note header
+- The right-click context menu for files in the file explorer
+
+The command copies an `obsidian://open` URL to your clipboard that points to the note.
+
+## Common parameters
+
+The following parameters are used across multiple URI actions. Understanding these common parameters will help you construct URIs for different actions more easily.
+
+### `vault`
+
+Specifies which vault to target. Can be either the vault name or the vault ID[^1].
+
+- **Vault name**: The human-readable name of your vault (e.g., `my vault`).
+- **Vault ID**: A unique 16-character identifier for the vault (e.g., `ef6ca3e3b524d22f`). Using the vault ID is more reliable if you have multiple vaults with similar names or if you rename your vault.
```

Comment:
In the same vein, I'd add to this that it also can't accidentally fail due to the percent encoding!